service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/docmosis/OrderCreationServiceDocmosisTest.java

```diff
@@ -65,6 +68,8 @@
     C23EPOAdditionalDocumentsCollector.class,
     C35bISODocumentParameterGenerator.class,
     OrderDetailsWithEndTypeGenerator.class,
+    UploadedOrderDocumentGenerator.class,
+    DocumentSealingService.class,
```

Comment:
very minor but I notice that the `documentDownloadService` isn't actually used, you can annotate the class with `@MockBeans({@MockBean(DocumentDownloadService.class)})` to remove the unused field

Reply:
👍